bellatrix.playwright/src/main/java/solutions/bellatrix/playwright/infrastructure/ExecutionType.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2024 Automate The Planet Ltd.
+ * Author: Miriam Kyoseva
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package solutions.bellatrix.playwright.infrastructure;
+
+public enum ExecutionType {
+    REGULAR,
```

Comment:
We should discuss with Anton but I think a better name for this mode is `LOCAL` and not `REGULAR`